Sources/PostmarkClient/Provider.swift

```diff
@@ -0,0 +1,47 @@
+import Vapor
+import HTTP
+
+
+public final class Provider: Vapor.Provider {
```

Comment:
This is pretty much fine. On line 17, `sg` was called `sg` as shorthand for Sendgrid. You might use `pm` or `postmark` or `postmarkConfiguration` as you see fit.

On line 32, we can now uncomment these lines because I have added an extension to `Droplet`.